libs/watt/src/lib/components/breadcrumbs/watt-breadcrumbs.component.ts

```diff
@@ -54,14 +51,14 @@ export class WattBreadcrumbComponent {
   styleUrls: ['./watt-breadcrumbs.component.scss'],
   template: `
     <nav>
-      @for (breadcrumb of breadcrumbs; track breadcrumb; let isLast = $last) {
+      @for (breadcrumb of breadcrumbs(); track breadcrumb; let isLast = $last) {
         <span
           class="watt-breadcrumb"
           (click)="breadcrumb.click.emit($event)"
-          [class.interactive]="breadcrumb.click.observed"
-          [attr.role]="breadcrumb.click.observed ? 'link' : null"
+          [class.interactive]="breadcrumb.click"
+          [attr.role]="breadcrumb.click ? 'link' : null"
```

Comment:
question: doesn't this always set `link` `role`/`interactive` class on the element?

Reply:
Think you are right, buuuut https://github.com/angular/angular/issues/54837

Reply:
It should be fixed now :-)